hibernate-core/src/main/antlr/sql-gen.g

```diff
@@ -191,18 +191,15 @@ whereClauseExpr
 	| booleanExpr[ false ]
 	;
 
-orderBySqlFragmentOrExprs
-	: sqlToken // for the purpose of mapping-defined orderBy SQL fragment
-	| orderExprs
-	;
-
 orderExprs { String ordExp = null; String ordDir = null; String ordNul = null; }
 	// TODO: remove goofy space before the comma when we don't have to regression test anymore.
 	// Dialect is provided a hook to render each ORDER BY element, so the expression is being captured instead of
 	// printing to the SQL output directly. See Dialect#renderOrderByElement(String, String, String, NullPrecedence).
```

Comment:
so the above comment only applies in HQL. Now we allow for another possibility that "printing to the SQL output directly". We need to add more explanation to the comment above, for it is not accurate any more.